PyRDF/Operation.py

```diff
@@ -22,6 +49,7 @@ def _classify_operation(self, name):
         Method to classify the given
         operation as action or 
         transformation
+
         """
 
```

Comment:
It does not apply to this PR but: where are my requested `is_action` and `is_transformation` methods plus their corresponding use throughout the rest of the code to make it simpler? :smile: 
Can you make another PR with those changes?

Reply:
@etejedor , oh yes. I'd created those in #14 , which I'd closed to split the commits. Forgot to make a separate PR during that time. Will one now surely :)